### PR TITLE
feat(connector): [WELLSFARGO_PAYOUT] PR template code

### PR DESCRIPTION
**Pull request by awasthi21**
## Type of Change

- [ ] Bugfix
- [x] New feature
- [ ] Enhancement
- [ ] Refactoring
- [ ] Dependency updates
- [ ] Documentation
- [ ] CI/CD

## Description
Template PR for Wellsfargo Payout

### Additional Changes

- [x] This PR modifies the API contract
- [ ] This PR modifies the database schema
- [x] This PR modifies application configuration/environment variables

## Checklist

- [x] I formatted the code `cargo +nightly fmt --all`
- [x] I addressed lints thrown by `cargo clippy`
- [x] I reviewed the submitted code
- [ ] I added unit tests for my changes where possible

### CODE REVIEW

File: crates/router/src/connector/braintree/transformers.rs [hunk 1]
```diff
@@ -1203,12 +1203,6 @@ pub struct CaptureInputData {
     transaction: CaptureTransactionBody,
 }
 
-#[derive(Debug, Clone, Serialize)]
```
Comment (awasthi21) [inline crates/router/src/connector/braintree/transformers.rs:1206]: Dead Code

Comment (deepanshu-iiitu) [reply crates/router/src/connector/braintree/transformers.rs:1206]: Remove this change. Only relevant changes for adding WellsFargoPayout template should be included in this PR.

File: crates/router/tests/connectors/sample_auth.toml [hunk 2]
```diff
@@ -257,3 +257,5 @@ api_secret = "Secret key"
 
 [paybox]
 api_key="API Key"
+
+[wellsfargopayout]
```
Comment (deepanshu-iiitu) [inline crates/router/tests/connectors/sample_auth.toml:261]: Add `api_key="API Key"` here

**Review by deepanshu-iiitu — CHANGES_REQUESTED** [review]

**Review by SamraatBansal — APPROVED** [review]

**Review by sai-harsha-vardhan — DISMISSED** [review]

File: config/deployments/production.toml [hunk 3]
```diff
@@ -94,6 +94,7 @@ trustpay.base_url_bank_redirects = "https://aapi.trustpay.eu/"
 tsys.base_url = "https://gateway.transit-pass.com/"
 volt.base_url = "https://api.volt.io/"
 wellsfargo.base_url = "https://api.cybersource.com/"
+wellsfargopayout.base_url = "https://api.wellsfargo.com/"
```
Comment (SanchithHegde) [inline config/deployments/production.toml:97]: Add this URL in sandbox, integration_test TOML files as well.

**Review by SanchithHegde — DISMISSED** [review]

**Review by SanchithHegde — APPROVED** [review]

**Review by sai-harsha-vardhan — APPROVED** [review]

**Review by deepanshu-iiitu — APPROVED** [review]

**Pull request merged**